### PR TITLE
[Doc release] Update Triage policy in Contributing doc

### PR DESCRIPTION
**Pull request by pixelhandler**
- Adds info for status / priority labels
- Adds policy for stale/abandonned issues

### CODE REVIEW

**Comment by pixelhandler** [comment]
@emberjs/issues-team @emberjs/learning-team-managers @emberjs/core What do you think about the addition of status and priority labels? Also how about a policy regarding stale or abandoned issues?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -56,6 +56,14 @@ original notice.
 * If you submit a feature request as an issue, you will be invited to follow the
 [instructions in this document](https://github.com/emberjs/ember.js/blob/master/CONTRIBUTING.md#requesting-a-feature)
 and the issue will be closed
+* Issues that become stale (or abandonned) will be labelled accordingly
+  to inform the original poster and ember contributors that the issue
+  should be closed since the issue is no longer actionable. The issue
+  can be reopened at a later time if needed, e.g. becomes actionable again.
+* If possible issues will be labelled to indicate the status or priority.
+  For example, labels may have a prefix for `Status: X`, or `Priority: X`.
+  Statuses may include: `In Progess`, `On Hold`. Priorities may include:
```
Comment (acorncom) [inline CONTRIBUTING.md]: *In Progress*

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -56,6 +56,14 @@ original notice.
 * If you submit a feature request as an issue, you will be invited to follow the
 [instructions in this document](https://github.com/emberjs/ember.js/blob/master/CONTRIBUTING.md#requesting-a-feature)
 and the issue will be closed
+* Issues that become stale (or abandonned) will be labelled accordingly
```
Comment (acorncom) [inline CONTRIBUTING.md]: *abandoned*

**Comment by acorncom** [comment]
Never mind the typo fixes, apparently I can just get in and do them myself! ;-)  Yes, that looks reasonable to me 👍

**Comment by rwjblue** [comment]
@pixelhandler - Seems good to me, I'll add to next agenda to see if anyone has objections.

**Comment by locks** [comment]
@pixelhandler I'd like to note this shouldn't have `[DOC x]` because it doesn't need to get cherrypicked into release branches

**Comment by pixelhandler** [comment]
@locks what should the title have?

**Comment by pixelhandler** [comment]
@rwjblue thanks let me know you think after the team discusses. :)

**Comment by pixelhandler** [comment]
@emberjs/issues-team once this merges - we can add the **"Inactive"** label and create a (shared) comment which links back (to this section in the contributing file) and explains what "Inactive" means as well as informs the original poster regarding timing to close out the issue.

**Comment by pixelhandler** [comment]
@rwjblue were you able to discuss with the team?

**Review by rwjblue — APPROVED** [review]

**Pull request merged**